services/ml/compilation_impl.h

```diff
@@ -32,6 +32,13 @@ struct ElementWiseParams {
 };
 
 struct ConvParams {
+  int32_t input_type;
+  int32_t output_type;
+  int32_t filter_type;
+  int32_t bias_type;
+  int32_t output_mask;
+  uint32_t output_scales_count;
+  float* output_scales;
```

Comment:
If they are only used by dnnl, it would be better to move to dnnl specific structures.